### DISCUSSION TITLE
Is the write_srt no longer available?

### DISCUSSION THREAD

**Discussion by alexprinz22** [openai/whisper#1289] [Q&A]
Is the write_srt no longer available? 

i dont see this class in utils.py?

how can I replace it?

**Comment by phineas-pta** [comment]
this change has happened for several months, you can do: 

```python
writer = whisper.utils.get_writer("srt", "YOUR FOLDER")
writer(result, "YOUR FILE.srt")
```

**Reply by sovdevs** [reply]
#req.txt
openai-whisper @ git+https://github.com/openai/whisper.git@b91c907694f96a3fb9da03d4bbdc83fbcd3a40a4

#main.py
import whisper
 .... 
 writer = whisper.utils.get_writer("srt", output_dir)
             ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'whisper.utils' has no attribute 'get_writer'

**Reply by phineas-pta** [reply]
`pip install openai-whisper`
